packages/core/saved-objects/core-saved-objects-api-server/index.ts

```diff
@@ -69,4 +69,4 @@ export type {
   SavedObjectAttributes,
   SavedObjectAttributeSingle,
   SavedObjectReference,
-} from '@kbn/core-saved-objects-common/src/server_types';
```

Comment:
You're importing the client-side, deprecated versions by doing so.

Reply:
Nice catch, will rollback these changes!